transaction/core/src/ring_signature/error.rs

```diff
@@ -42,10 +41,6 @@ pub enum Error {
     #[fail(display = "There was an opaque error returned by another crate or library")]
     InternalError,
 
-    /// Public keys must be valid Ristretto points.
-    #[fail(display = "KeyError")]
-    KeyError,
```

Comment:
This enum contains another error variant for RistrettoPoints that do not compress/decompress (i.e. InvalidCurvePoint).